WalletWasabi.Fluent/ViewModels/AddWallet/HardwareWallet/ConnectHardwareWalletViewModel.cs

```diff
@@ -142,7 +142,7 @@ private void EvaluateDetectionResult(HwiEnumerateEntry[] devices, CancellationTo
 	{
 		if (devices.Length == 0)
 		{
-			Message = "Connect your wallet to the USB port on your PC / Enter the PIN on the Wallet.";
+			Message = "Connect the hardware wallet to the PC & enter the PIN on the device.";
```

Comment:
Using `&` is not correct. 

The previous message contained `/` because we cannot detect/distinguish that two cases. Either there is no device connected OR there is a device connected, but it must be unlocked with the PIN.
It is not like "connect your device AND unlock it with the PIN", because there might not be a PIN at all.

The sad thing is everything depends on the device... there are devices where we can detect the case above and there are ones where we don't.

If you really want to improve the message here, then try to find something general... because when this message comes you cannot tell what is the exact problem.